### DISCUSSION TITLE
return erro when import ResNetFeatures

### DISCUSSION THREAD

**Discussion by 2345678y3** [Project-MONAI/MONAI#7972] [General]
when I  from monai.networks.nets import ResNetFeatures   the erro is:
ImportError: cannot import name 'ResNetFeatures' from 'monai.networks.nets' (/root/miniconda3/envs/nnunetv2/lib/python3.9/site-packages/monai/networks/nets/__init__.py)

**Comment by 2345678y3** [comment]
![Uploading 屏幕截图 2024-07-27 233451.png…]()

**Comment by KumoLiu** [comment]
Hi @2345678y3, may I know your MONAI version, it's a new feature be included since 1.3.1. Thanks.

**Comment by 2345678y3** [comment]
> Hi @2345678y3, may I know your MONAI version, it's a new feature be included since 1.3.1. Thanks.

thanks for your answer,  i have upgrade vision,but there happend new error  
![屏幕截图 2024-07-31 002514](https://github.com/user-attachments/assets/d47a5e7c-d523-427e-9bcd-9376cc9a73ed)

**Comment by KumoLiu** [comment]
Can not reproduce the issue, please try latest MONAI and also ensure there is only one monai installed.
```
>>> x = ResNetFeatures("resnet18", True, 3, 1)
resnet_18.pth: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████| 132M/132M [00:10<00:00, 12.4MB/s]
```

Hope it helps, thanks.
(Move to discussion for now, more like a usage issue)
